man/timesyncd.conf.xml

```diff
@@ -101,6 +101,36 @@
         is used instead.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MinPollIntervalSec=</varname></term>
+        <term><varname>MaxPollIntervalSec=</varname></term>
+        <listitem><para>The minimum and maximum poll intervals for NTP messages.
+        Take time (in seconds). <varname>MinPollIntervalSec=</varname> must be larger than 16 sec.
+        <varname>MaxPollIntervalSec=</varname> must be larger than <varname>MinPollIntervalSec=</varname>.
+        <varname>MinPollIntervalSec=</varname> defaults to 32 seconds, and
+        <varname>MaxPollIntervalSec=</varname> defaults to 2048 seconds.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Accuracy=</varname></term>
+        <listitem><para>Expected Accuracy of time synchronization in seconds, which is used to adjust the
+        poll interval. Takes a positive floating point value. Defaults to 0.2.</para></listitem>
```

Comment:
if the unit is seconds, then this should be called AccuracySec= and we should parse this using our usual time parsers (even if this is stored in a floating point value ultimately internally, what matters is that time values are always exposed the same way in a uniform fashion)